src/react-elastic-carousel/components/styled/Slider.js

```diff
@@ -51,7 +51,7 @@ export default styled.div`
   position: absolute;
   display: flex;
   flex-direction: ${({ verticalMode }) => (verticalMode ? "column" : "row")};
-  min-height: 100%;
+  ${({ verticalMode }) => (verticalMode ? "min-height: 100%;" : "")}
```

Comment:
can you show an example of what it solves exactly?

Reply:
I can explain at a high-level to start. If you want a repro case, let me know.

Let's say we had a carousel slide with 4 rows of text on top of each other. If we were to remove one of those rows of text, we would expect that carousel's height to decrease, as the slides inside have gotten shorter. However, we're setting the height of the carousel container based on the initial size on the slide container (provided by the ResizeObserver), and setting this Slider's `min-height` to be 100%. This means that when the element's contents would result in the Slide getting shorter, the combo of the explicit height set on the container + the `min-height: 100%` rule means this Slider element expands up to the explicit height set, even if its contents are shorter than that height.

It looking through the demos on the site, I don't see this rule negatively impacting anything, which makes sense, cuz it appears to be specifically needed for `verticalMode`, hence the conditional.

Happy to throw together a demo that demonstrates this if needed.

I will say, on our end, we don't have quite this scenario, and I haven't been able to track down _why_ our initial render produces a `ResizeEntry` with a height larger than the actual contents, but with this change, once the contents have completely finished rendering, everything is the correct height, as the RO will fire a second event with the final position. This does explain conceptually what's going on though, and I still think this is a bug cuz it would mean if you swapped out the contents on the carousel with any smaller contents, the carousel wouldn't adjust accordingly.